core/src/main/scala/kafka/log/LogValidator.scala

```diff
@@ -175,12 +175,6 @@ private[kafka] object LogValidator extends Logging {
                                                  messageFormatVersion: Byte = RecordBatch.CURRENT_MAGIC_VALUE,
                                                  messageTimestampType: TimestampType,
                                                  messageTimestampDiffMaxMs: Long): ValidationAndOffsetAssignResult = {
-      // Deal with compressed messages
```

Comment:
The steps are in the method scaladoc still, so I think it's OK to keep them. What do you think?